### ISSUE TITLE
minikube –vm-driver=xhyve start hangs

### ISSUE THREAD

**Issue by iceycake** [kubernetes/minikube#1665]
**BUG REPORT** (choose one):

**Minikube version** (use `minikube version`):
0.20.0

**Environment**:
- **Mac OS Sierra 10.12.5**
- **VM Driver** xhyve 0.3.3
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`): n/a
- **Install tools**: brew
- **Others**:

**What happened**:
Hung at Starting VM...

**Comment by r2d4** [comment]
Does `$ minikube logs` show anything?  If not, could you try to delete the VM and then run `minikube start` with the `--v 10 --alsologtostderr` flags?

**Comment by chrishiestand** [comment]
I can't reproduce this from a fresh install with the same environment as above.

I also did this to fix another issue (I did not care about retaining any data here): `rm -rf ~/.minikube/`

```$ minikube start --vm-driver xhyve
Starting local Kubernetes v1.6.4 cluster...
Starting VM...
Downloading Minikube ISO
 90.95 MB / 90.95 MB [==============================================] 100.00% 0s
Moving files into cluster...
Setting up certs...
Starting cluster components...
Connecting to cluster...
Setting up kubeconfig...
Kubectl is now configured to use the cluster.```

**Comment by iceycake** [comment]
Removing ~/.minikube makes sense.  I could have old configuration somewhere as I can’t reproduce the problem on my other Mac.

**Comment by iceycake** [comment]
Finally, I have time to continue troubleshoot this problem.  I deleted ~/.minikube and start everything from scratch again.  Here is the detail log

```
mbp15-andychan:~ andy.chan$ minikube start --vm-driver=xhyve --logtostderr --show-libmachine-logs --v=5
W0726 10:28:03.395814    9006 root.go:153] Error reading config file at /Users/andy.chan/.minikube/config/config.json: open /Users/andy.chan/.minikube/config/config.json: no such file or directory

--show-libmachine-logs is deprecated.
Please use --v=3 to show libmachine logs, and --v=7 for debug level libmachine logs

I0726 10:28:03.396322    9006 notify.go:112] Checking for updates...
There is a newer version of minikube available (v0.21.0).  Download it here:
https://github.com/kubernetes/minikube/releases/tag/v0.21.0

To disable this notification, run the following:
minikube config set WantUpdateNotification false
Starting local Kubernetes v1.6.4 cluster...
Starting VM...
Downloading Minikube ISO
 90.95 MB / 90.95 MB [==============================================] 100.00% 0s
Creating CA: /Users/andy.chan/.minikube/certs/ca.pem
Creating client certificate: /Users/andy.chan/.minikube/certs/cert.pem
Running pre-create checks...
Creating machine...
(minikube) Downloading /Users/andy.chan/.minikube/cache/boot2docker.iso from file:///Users/andy.chan/.minikube/cache/iso/minikube-v0.20.0.iso...
(minikube) Creating VM...
(minikube) /dev/disk6          	                               	/Users/andy.chan/.minikube/machines/minikube/b2d-image
(minikube) "disk6" unmounted.
(minikube) "disk6" ejected.
(minikube) Generating 20000MB disk image...
(minikube) created: /Users/andy.chan/.minikube/machines/minikube/root-volume.sparsebundle
(minikube) Creating SSH key...
(minikube) Fix file permission...
(minikube) Generate UUID...
(minikube) Convert UUID to MAC address...
(minikube) Starting minikube...
(minikube) Waiting for VM to come online...
(minikube) Waiting on a pseudo-terminal to be ready... done
(minikube) Hook up your terminal emulator to /dev/ttys004 in order to connect to your VM
```

It stops there and never continue.  I'm wondering if it is related to Cisco Anywhere VPN since everything works fine without starting the VPN.  Any idea how to troubleshoot it?

**Comment by r2d4** [comment]
@iceycake Theres a pretty long discussion about the Cisco Anywhere VPN with minikube here https://github.com/kubernetes/minikube/issues/1099.

I know it closed, but do you mind if we track workarounds there?  I'm not sure if theres much we can do from the minikube side, but you might find something that works for you there.
